docs/topics/asyncio.rst

```diff
@@ -98,6 +98,52 @@ Futures. Scrapy provides two helpers for this:
          into your own code.
 
 
+Async additional requests
+=========================
+
+The spider below shows a single use-case of scraping a page and gathering a price from a separate URL::
+
+
+    class SingleRequestSpider(scrapy.Spider):
+        name = "single"
+        start_urls = ["https://example.org/product"]
+
+        async def parse(self, response, **kwargs):
+            additional_request = scrapy.Request('https://example.org/price')
+            deferred = self.crawler.engine.download(additional_request)
+            additional_response = await maybe_deferred_to_future(deferred)
+            yield {
+                'h1': response.css('h1').get(),
+                'price': additional_response.css('#price').get(),
+            }
+
+
+The spider gathering batch requests::
```

Comment:
I would avoid using the word “batch” here, as I think it can be confusing. To me, batch requests means “send a batch of requests, wait for all of them, then send another batch, wait again”, which is not exactly what we are featuring here.

```suggestion
You can also send multiple requests in parallel:

.. code-block:: python
```